pkg/controller/core.oam.dev/v1alpha2/application/application_controller.go

```diff
@@ -247,46 +271,46 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		klog.Info("Finished rollout ")
 	}
 
-	app.Status.Phase = common.ApplicationHealthChecking
-	klog.Info("Check application health status")
-
-	// check application health status if no health check policy is applied
-	if !hasHealthCheckPolicy(appFile.Policies) {
-		appCompStatus, healthy, err := handler.aggregateHealthStatus(appFile)
-		if err != nil {
-			klog.ErrorS(err, "application", klog.KObj(app))
-			r.Recorder.Event(app, event.Warning(velatypes.ReasonFailedHealthCheck, err))
-			app.Status.SetConditions(condition.Condition{
-				Type:               v1beta1.TypeHealthy,
-				Status:             corev1.ConditionFalse,
-				LastTransitionTime: metav1.Now(),
-				Reason:             v1beta1.ReasonHealthCheckErr,
-			})
-			return r.endWithNegativeCondition(ctx, app, condition.ErrorCondition("HealthCheck", err))
-		}
-		app.Status.Services = appCompStatus
-		if !healthy {
+	// Keep the ability to get status
+	if !appFile.SkipHealthCheck {
+		// check application health status if no health check policy is applied
+		if !hasHealthCheckPolicy(appFile.Policies) {
```

Comment:
1. please merge it with the `skipHealthCheck` condition. 
2.if the healthcheck is skipped, who will update the status?